src/qibolab/platforms/dummy.py

```diff
@@ -63,3 +68,133 @@ def set_current(self, qubit, current):  # pragma: no cover
 
     def set_gain(self, qubit, gain):  # pragma: no cover
         pass
+
+    def get_attenuation(self, qubit):  # pragma: no cover
+        pass
+
+    def get_current(self, qubit):  # pragma: no cover
+        pass
+
+    def get_gain(self, qubit):  # pragma: no cover
+        pass
```

Comment:
I know you are missing coverage without the pragma, but there is a better option:
```suggestion
    def get_gain(self, qubit):
        """Here explain why this function is empty"""
```

If it has a docstring, you do not need to add `pass`, and an explanatory docstring is more useful than `# pragma: no cover` + `pass`, even with the same amount of characters.

Reply:
Fixed